libr/cons/d/basic

```diff
@@ -36,7 +36,7 @@ ec call cyan
 ec cjmp green
 ec jmp green
 ec pop yellow
-ec line_highlight #00003f
+ec linehl #00003f
```

Comment:
switch to "rgb:" maybe?

Reply:
okay d111dea5bc77b3f51b7c3d9a2f0349739339d48a